test/tokenCache.js

```diff
@@ -0,0 +1,34 @@
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let server = require('../server');
+
+chai.use(chaiHttp);
+
+describe('Token Cache', () => {
+  describe('API', () => {
+    it('it should GET token info', function (done) {
+      chai.request(server).get('/tokenCache/tokenInfo').end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.a('object');
```

Comment:
can we check on some properties here? to also verify that there is a proper response

Reply:
I'll check